_data/conferences.yml

```diff
@@ -81,3 +81,38 @@
   speaker: Lori Lalonde
   location: Kitchener, ON
   topic: Windows 10 Developer Camp
+
+- date: 2015-10-19
+  event: Winnipeg .NET User Group
+  link: http://www.winnipegdotnet.org
+  speaker: James Chambers
+  location: Winnipeg, MB
+  topic: What to Expect with MVC 6
+
+- date: 2015-11-09
+  event: Microsoft Virtual Academy
+  link: http://www.microsoftvirtualacademy.com
+  speaker: James Chambers
+  location: Seattle, WA
+  topic: "Applications on Azure: Putting all the Pieces Together"
+  
+- date: 2015-11-09
+  event: Microsoft Virtual Academy
+  link: http://www.microsoftvirtualacademy.com
+  speaker: Simon Timms
+  location: Seattle, WA
+  topic: "Applications on Azure: Putting all the Pieces Together"
+  
+- date: 2016-02-24
+  event: ConFoo
+  link: http://confoo.ca
+  speaker: Lori Lalonde
+  location: Montreal, QC
+  topic: Cross Platform Localization Strategies
+
+- date: 2016-02-24
+  event: ConFoo
+  link: http://confoo.ca
+  speaker: James Chambers
+  location: Montreal, QC
+  topic: Success as a Remote Worker: Love your Job, Have a Life"
```

Comment:
Success as a Remote Worker: Love Your Job, Have a Life

Also, you're missing the initial quote.